src/transformers/models/layoutlmv3/modeling_layoutlmv3.py

```diff
@@ -118,6 +120,11 @@
             Whether or not to return a [`~utils.ModelOutput`] instead of a plain tuple.
 """
 
+LAYOUTLMV3MODEL_INPUTS_DOCSTRING = r"""
```

Comment:
```suggestion
LAYOUTLMV3_INPUT_CLARIFICATION = r"""
```
Maybe this could be a better name :D